backend/services/inventory/Makefile

```diff
@@ -1,3 +1,24 @@
 COMPONENT := inventory
 
+override TESTFLAGS := $(TESTFLAGS) -p 1
+
+DOCFILES := $(wildcard ../../../backend/docs/api/inventory/*.yaml)
+MAKEDIR := $(shell dirname $(realpath $(firstword $(MAKEFILE_LIST))))
+
+tests/%: ../../../backend/docs/api/inventory/%.yaml
+	[ -e $@ ] && rm -r $@; \
+	docker run --rm -t -v $(MAKEDIR):/work -v $(MAKEDIR)/../../../backend/docs/api:/api -w /work \
+		--ulimit nofile=65536:65536 \
+		-u $(shell id -u):$(shell id -g) \
+		openapitools/openapi-generator-cli:v4.3.1 generate \
```

Comment:
I'm not going to argue about the usage of this image, just making a remark that it's more than 5 years old and 2 major versions behind latest. :memo:
https://github.com/OpenAPITools/openapi-generator/releases/tag/v4.3.1